ProductService.js

```diff
@@ -0,0 +1,110 @@
+class Product {
+    constructor(name, destription, price, tags, images) {
+        this.name = name;
+        this.destription = destription;
+        this.price = price;
+        this.tags = tags;
+        this.images = images;
+        this.favoriteCount = 0;
+    }
+    favorite() {
+        this.favoriteCount++;
+    }
+}
+
+class ElectronicProduct extends Product {
+    (name, destription, price, tags, images, manufacturer) {
+        super (name, destription, price, tags, images);
+        this.manufacturer = manufacturer;
+    }
+}
+
+const BASE_URL = 'https://panda-market-api-crud.vercel.app';
+
+async function getProductList (page = 1, pageSize = 10, keyword = '') {
+    try {
+        const response = await fetch (`${BASE_URL}/products?page=${page}&pageSize=
+            ${pageSize}&keyword=${keyword}`);
+        if (!response.ok) {
+            throw new Error (`HTTP error! status: ${response.status}`);
+        }
+        const data = await response.json();
+        const products = data.map (item => {
```

Comment:
함수 뒤에 스페이스를 넣어서 사용하는 경향이 있어요. 대부분은 붙여서 사용하기때문에 붙여서 사용해주세요.

```suggestion
        const products = data.map(item => {
```